### ISSUE TITLE
(Voicing) Is the focus order correct? I think it changed.

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#285]
The keyboard focus order changed. It used to start from the visual top of the sim with the Force Arrow reading blocks. It now starts with Move mass 1/2 position sliders.

Old keyboard focus order order:
- Force Arrow reading blocks
- Sphere position sliders
- Mass controls

Now keyboard focus order is :
- Sphere position sliders
- Force Arrow reading blocks
- Mass controls

I don't remember requesting this change, and it seems a bit odd to me.

Can we have the keyboard focus order back to the old order?	

The distance and mass reading blocks are not accessible to keyboard users.

**Comment by terracoda** [comment]
Added bug and question labels because I am not sure if we intentionally changed the focus order or not.

**Comment by jessegreenberg** [comment]
This has been fixed in the above commit.

**Comment by jessegreenberg** [comment]
Here is a new version with this, @terracoda can you please verify? https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-dev.32/phet/gravity-force-lab-basics_en_phet.html?voicingSurvey=true

**Comment by terracoda** [comment]
Fixed :-)
